sleap/util.py

```
@@ -1,32 +1,52 @@
-"""A miscellaneous set of utility functions. 
+"""A miscellaneous set of utility functions.
 
 Try not to put things in here unless they really have no other place.
 """
 
+from __future__ import annotations
+
+import base64
```

Ruff

`base64` imported but unused

Remove unused import: `base64`

(F401)